### ISSUE TITLE
NG+ animation bugs

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#504]
Since installing the community patch in a fresh new game, I experienced uncontrolled bouncing when on mine lifts and uncontrolled involuntary zoom movements in outpost build-flyover mode. But only in NG+ This is the _only_ new mod I added before starting the new game, so something adverse happened because of it. It's an unproven theory, but I don't know what else can be causing it since this is the only mod I added before starting my new game.

---

## Details 
I'm still on community patch 0.0.1. I leave my mods alone when in a game. Maybe this is resolved in 0.0.2? The bug isn't enough to make me quit my game, but it is incredibly annoying. 

## Game Version
1.7.36

## Language
English (EN)

## New Game
No

## Reproduction Steps
I'm still playing in the same game. Just play an honest game and get to NG+ and try building an outpost or descending on a mining lift. Like I said, I'm not going to ditch my current game -- as annoying as the bugs are -- if I'm not assured staring a fresh new game is going to solve anything.

**Comment by Constellation-VASCO** [comment]
OK, so I deleted all my saves and started a new game where the only change was that I removed this mod before starting the new game. The symptoms I reported went away. So from my experience, this mod created those symptoms. Why? I don't know. If there's some kind of under-the-hood log I can give you, I'm more than happy to share. But for now my workaround is not to have this mod installed.
